book/types_of_data.md

```diff
@@ -209,35 +209,45 @@ Structured data builds from the simple data. For example, instead of a single in
 
 ## Records
 
-Records hold key-value pairs, which associate string keys with various data values. Record syntax is very similar to objects in JSON. However, commas are _not_ required to separate values if Nushell can easily distinguish them!
+A record is a collection of key-value pairs. Unlike most languages, comma separators are optional.
 
 ```nu
-> {name: sam rank: 10}
+> {name: sam, rank: 10}
```

Comment:
I prefer no commas in examples. Showing them tries to take away one of the superpowers of nushell, IMO.